ICGE-Simulation/src/main/java/de/unistuttgart/informatik/fius/icge/simulation/internal/playfield/StandardPlayfield.java

```diff
@@ -54,6 +54,37 @@ public class StandardPlayfield implements Playfield {
     
     private Consumer<List<Drawable>> drawablesChangedListener;
     
+    private long             lastEntityDraw            = 0;
+    private volatile boolean delayedEntityDraw         = false;
```

Comment:
```suggestion
    private volatile boolean awaitingEntityDraw        = false;
```
With this name, the `if` in `drawEntities` is easier to understand